.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Maplibre-Swift-Macros
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: macos-14
+    strategy:
+      matrix:
+        scheme: [
+          MapLibreSwiftMacros
+        ]
+        destination: [
+          'platform=watchOS Simulator,name=Apple Watch Ultra 2 (49mm)',
```

Comment:
Wait, does MapLibre even support watchOS? 🤔

Reply:
Lol, fair enough.

We do want to merge this so we can start publishing tagged releases to fix: https://github.com/stadiamaps/maplibre-swiftui-dsl-playground/issues/24.